app/cloud/server/index.js

```diff
@@ -34,6 +37,22 @@ Meteor.startup(function() {
 			job: syncWorkspace,
 		});
 	});
+
+	const { workspaceRegistered } = retrieveRegistrationStatus();
+
+	if (process.env.REG_TOKEN && process.env.REG_TOKEN !== '' && !workspaceRegistered) {
```

Comment:
Maybe we can change this to test for negative to avoid nesting the if 👀

Reply:
@KevLehman can you make a suggestion here with an example of what you mean?

Reply:
It's something like:
```
if (!process.env.REG_TOKEN || process.env.REG_TOKEN.trim() === '' || workspaceRegistered) return;
try { ... }
```